.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+repos:
+    - repo: local
```

Comment:
I tried to use a pre-commit mirror for this instead of a local script, but the maintainer literally archived the mirror [one hour ago](https://github.com/pre-commit/mirrors-prettier), which is the luckiest timing in the world.